### PR TITLE
add link to signup page

### PR DESCRIPTION
**Pull request by yury-rubanenko**

### CODE REVIEW

File: src/app/(auth)/login/page.js [hunk 1]
```diff
@@ -52,6 +53,11 @@ export default function Login() {
             <Button type="submit" variant="contained" color="green" fullWidth sx={{ mt: 6 }}>
               Login
             </Button>
+            <Typography sx={{ mt: 2, textAlign: 'center' }}>
+              <Link href="/signup" style={{ marginLeft: '5px', fontWeight: 'bold', color: 'green' }}>
```
Comment (maryna-velychko) [inline src/app/(auth)/login/page.js:57]: Use sx={{}} for style. Why you need left margin?
Also it should be a link to sign in page at a sign up page

File: src/app/(auth)/login/page.js [hunk 2]
```diff
@@ -52,6 +53,11 @@ export default function Login() {
             <Button type="submit" variant="contained" color="green" fullWidth sx={{ mt: 6 }}>
               Login
             </Button>
+            <Typography sx={{ mt: 2, textAlign: 'center' }}>
```
[the remark this answers is not in the document]
Comment (yury-rubanenko) [reply src/app/(auth)/login/page.js:56]: ok

File: src/app/(auth)/signup/components/SignUpAccount.js [hunk 3]
```diff
@@ -89,9 +89,12 @@ export default function SignUpAccount({ goToNextStep }) {
               }
               sx={{ mt: 6 }}
             />
-            <Button type="submit" variant="contained" color="grey" fullWidth sx={{ mt: 1 }}>
+            <Button type="submit" variant="contained" color="grey" fullWidth sx={{ mt: 1, mb: 2 }}>
               Sign up with email
             </Button>
+              <Link href="/login" sx={{ textAlign: 'center', fontWeight: 'bold', color: 'primary.main' }}>
```
Comment (maryna-velychko) [inline src/app/(auth)/signup/components/SignUpAccount.js:95]: the same

File: src/app/(auth)/login/page.js [hunk 4]
```diff
@@ -49,9 +50,12 @@ export default function Login() {
             <FormikTextField name="email" label="Email Address" fullWidth sx={{ mt: 3 }} />
             <FormikTextField name="password" label="Password" type="password" fullWidth sx={{ mt: 1.5 }} />
             <FormikErrors sx={{ mt: 1.5 }} />
-            <Button type="submit" variant="contained" color="green" fullWidth sx={{ mt: 6 }}>
+            <Button type="submit" variant="contained" color="green" fullWidth sx={{ mt: 6, mb: 2 }}>
               Login
             </Button>
+              <Link href="/signup" sx={{ textAlign: 'center', fontWeight: 'bold', color: 'primary.main' }}>
```
Comment (maryna-velychko) [inline src/app/(auth)/login/page.js:56]: do you need this style here why fontWeight: 'bold', color: 'primary.main'? Now this style is still not applied in the function to which you pass them. Just delete it, it looks better without them to me

**Pull request merged**